pyproject.toml

```diff
@@ -15,11 +15,17 @@ repository="https://github.com/sicpa-dlab/aries-acapy-plugin-http-uniresolver"
 
 [tool.poetry.dependencies]
 python = "^3.6.9"
-aries-cloudagent = {git = "https://github.com/hyperledger/aries-cloudagent-python", rev = "main"}
 PyYAML = "~=5.4"
 ecdsa = "~=0.16.1"
 
 
+[tool.poetry.dependencies.aries-cloudagent]
+    git = "https://github.com/hyperledger/aries-cloudagent-python"
+    rev = "main"
+    # Temporarily use extras; these should go away once fixed in master
+    extras = ["indy", "bbs"]
+
+
```

Comment:
This shouldn't be necessary anymore... were there issues without making this change?

Reply:
It breaks on instantation:
```
Attaching to demo_resolver_1
resolver_1  | Traceback (most recent call last):
resolver_1  |   File "/home/indy/.pyenv/versions/3.6.13/lib/python3.6/runpy.py", line 193, in _run_module_as_main
resolver_1  |     "__main__", mod_spec)
resolver_1  |   File "/home/indy/.pyenv/versions/3.6.13/lib/python3.6/runpy.py", line 85, in _run_code
resolver_1  |     exec(code, run_globals)
resolver_1  |   File "/home/indy/.cache/pypoetry/virtualenvs/http-uniresolver-f-aLBW4O-py3.6/lib/python3.6/site-packages/aries_cloudagent/__main__.py", line 74, in <module>
resolver_1  |     main(sys.argv)
resolver_1  |   File "/home/indy/.cache/pypoetry/virtualenvs/http-uniresolver-f-aLBW4O-py3.6/lib/python3.6/site-packages/aries_cloudagent/__main__.py", line 71, in main
resolver_1  |     run(args)
resolver_1  |   File "/home/indy/.cache/pypoetry/virtualenvs/http-uniresolver-f-aLBW4O-py3.6/lib/python3.6/site-packages/aries_cloudagent/__main__.py", line 64, in run
resolver_1  |     run_command(command, args)
resolver_1  |   File "/home/indy/.cache/pypoetry/virtualenvs/http-uniresolver-f-aLBW4O-py3.6/lib/python3.6/site-packages/aries_cloudagent/commands/__init__.py", line 36, in run_command
resolver_1  |     module = load_command(command) or load_command("help")
resolver_1  |   File "/home/indy/.cache/pypoetry/virtualenvs/http-uniresolver-f-aLBW4O-py3.6/lib/python3.6/site-packages/aries_cloudagent/commands/__init__.py", line 31, in load_command
resolver_1  |     return import_module(module_path)
resolver_1  |   File "/home/indy/.pyenv/versions/3.6.13/lib/python3.6/importlib/__init__.py", line 126, in import_module
resolver_1  |     return _bootstrap._gcd_import(name[level:], package, level)
resolver_1  |   File "<frozen importlib._bootstrap>", line 994, in _gcd_import
resolver_1  |   File "<frozen importlib._bootstrap>", line 971, in _find_and_load
resolver_1  |   File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
resolver_1  |   File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
resolver_1  |   File "<frozen importlib._bootstrap_external>", line 678, in exec_module
resolver_1  |   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
resolver_1  |   File "/home/indy/.cache/pypoetry/virtualenvs/http-uniresolver-f-aLBW4O-py3.6/lib/python3.6/site-packages/aries_cloudagent/commands/start.py", line 16, in <module>
resolver_1  |     from ..core.conductor import Conductor
resolver_1  |   File "/home/indy/.cache/pypoetry/virtualenvs/http-uniresolver-f-aLBW4O-py3.6/lib/python3.6/site-packages/aries_cloudagent/core/conductor.py", line 35, in <module>
resolver_1  |     from ..protocols.out_of_band.v1_0.manager import OutOfBandManager
resolver_1  |   File "/home/indy/.cache/pypoetry/virtualenvs/http-uniresolver-f-aLBW4O-py3.6/lib/python3.6/site-packages/aries_cloudagent/protocols/out_of_band/v1_0/manager.py", line 16, in <module>
resolver_1  |     from ....indy.models.xform import indy_proof_req_preview2indy_requested_creds
resolver_1  |   File "/home/indy/.cache/pypoetry/virtualenvs/http-uniresolver-f-aLBW4O-py3.6/lib/python3.6/site-packages/aries_cloudagent/indy/models/xform.py", line 5, in <module>
resolver_1  |     from .pres_preview import IndyPresPreview
resolver_1  |   File "/home/indy/.cache/pypoetry/virtualenvs/http-uniresolver-f-aLBW4O-py3.6/lib/python3.6/site-packages/aries_cloudagent/indy/models/pres_preview.py", line 9, in <module>
resolver_1  |     from ...ledger.indy import IndySdkLedger
resolver_1  |   File "/home/indy/.cache/pypoetry/virtualenvs/http-uniresolver-f-aLBW4O-py3.6/lib/python3.6/site-packages/aries_cloudagent/ledger/indy.py", line 14, in <module>
resolver_1  |     import indy.ledger
resolver_1  | ModuleNotFoundError: No module named 'indy'

```

Reply:
Should be fixed by https://github.com/hyperledger/aries-cloudagent-python/pull/1236